typings.d.ts

```diff
@@ -1,5 +1,10 @@
 /// <reference path="./node_modules/typed-graphql/graphql.d.ts" />
 
+declare namespace NodeJS {
```

Comment:
Nope. By this declaration we are signalling TypeScript compiler that we have `browser` field in the `process`.